CGAL_ImageIO/include/CGAL/read_vtk_image_data.h

```diff
@@ -85,6 +85,8 @@ read_vtk_image_data(vtkImageData* vtk_image, Image_3::Own owning = Image_3::OWN_
   image->wdim = imageio_type.wdim;
   image->wordKind = imageio_type.wordKind;
   image->sign = imageio_type.sign;
+  if (!vtk_image->GetPointData() || !vtk_image->GetPointData()->GetScalars())
+    return Image_3();
```

Comment:
Memory leak. The fix might be:
```suggestion
  if (!vtk_image->GetPointData() || !vtk_image->GetPointData()->GetScalars()) {
    ::_freeImage(image);
    return Image_3();
  }
```